webapp/src/main/java/org/opentestsystem/rdw/admin/model/GroupSet.java

```diff
@@ -0,0 +1,46 @@
+package org.opentestsystem.rdw.admin.model;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Set;
+
+public class GroupSet {
```

Comment:
Had to create a wrapper which has the `writeable` property which indicates if the user is authorized to edit (and delete) this group.